src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -190,7 +195,8 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
             os,
             ports: debugConfiguration.dockerRun.ports,
             volumes: DockerDebugConfigurationProvider.inferVolumes(folder, debugConfiguration),
-            configureAspNetCoreSsl: ssl
+            configureAspNetCoreSsl: ssl,
+            configureDotNetUserSecrets: userSecrets,
```

Comment:
`userSecrets` doesn't look like a boolean based on the definition, while configureUserSecrets looks like one. Is there an implicit type-cast I'm missing here?

Reply:
I could rename it to configureDotNetUserSecrets, but userSecrets is just a yes/no inferred from the project file.